Translator.php

```diff
@@ -168,4 +175,27 @@ public function params()
     {
         return $this->_params;
     }
-} 
\ No newline at end of file
+
+    /**.
+     * @return array
+     */
+    public function filteredByFields():array
```

Comment:
yii2 uses public property instead of getters method

Reply:
made this method  same style like where(), params()

Reply:
it's difficult to undersand no have comments